prov/terraform/openstack/osinstance.go

```diff
@@ -414,28 +383,12 @@ func computeFloatingIPAddress(ctx context.Context, opts osInstanceOptions,
 	instanceName := opts.instanceName
 
 	log.Debugf("Looking for Floating IP")
-	var floatingIP string
-	resultChan := make(chan string, 1)
-	go func() {
-		for {
-			if fip, _ := deployments.GetInstanceCapabilityAttributeValue(ctx, deploymentID, networkNodeName, "0", "endpoint", "floating_ip_address"); fip != nil && fip.RawString() != "" {
-				resultChan <- fip.RawString()
-				return
-			}
 
-			select {
-			case <-time.After(1 * time.Second):
-			case <-ctx.Done():
-				// context cancelled, give up!
-				return
-			}
-		}
-	}()
-	select {
-	case floatingIP = <-resultChan:
-	case <-ctx.Done():
-		return ctx.Err()
+	floatingIP, err := deployments.LookupInstanceCapabilityAttributeValue(ctx, deploymentID, networkNodeName, deployments.DefaultInstanceName, "endpoint", "floating_ip_address")
```

Comment:
For floating ip addresses we will need to use `instanceName`, as there is one FIP per compute instance.

Reply:
Yes, I think you're right/ I fix it too